lib/classes/Service.js

```diff
@@ -46,6 +46,10 @@ class Service {
       return BbPromise.resolve();
     }
 
+    if (this.serverless.cli.isServiceIndependentRequest(this.serverless.processedInput)) {
+      return BbPromise.resolve();
+    }
```

Comment:
Sorry for picking on this now, but I've just realized, that we should actually do something else to be fully bulletproof.

Note that before this method is run, [plugin manager loads and parses config file](https://github.com/serverless/serverless/blob/5f85543d8e47a9afaec6811a57e1235e82949550/lib/Serverless.js#L75-L76). If there's any syntax error in config it'll still crash.

I believe full bulletproof solution would be to delete `this.serverless.config.servicePath` at this point https://github.com/serverless/serverless/blob/5f85543d8e47a9afaec6811a57e1235e82949550/lib/Serverless.js#L73

This will allow service independent commands to work exactly same as not in service context.